QgisModelBaker/libqgsprojectgen/dbconnector/db_connector.py

```diff
@@ -312,6 +312,14 @@ def create_basket(self, dataset_tid, topic):
         """
         return False, None
 
+    def get_tid_handling(self):
+        """
+        Returns if there is a tid handling enabled according to the settings table.
+        It's the tid handling that has been enabled "manually" (or by a plugin) and
+        not if t_ili_tids are used according the stable id defined in the model.
```

Comment:
Can this be rewritten a bit to leave no room for interpretation.
Something like

"Will return ``True`` if ``--createTidCol`` has been used to create the database.
... more information about model definition OID ...
"

please proof read